backend/drive/drive.go

```diff
@@ -532,6 +536,20 @@ Note that this detection is relying on error message strings which
 Google don't document so it may break in the future.
 `,
 			Advanced: true,
+		}, {
+			Name:     "service_account_folder",
+			Help:     "Use all files from the given folder as Service account file",
+			Advanced: true,
+		}, {
+			Name:     "account_disable_time",
+			Help:     "How long should an account be disabled when too many errors occur",
+			Value:    defaultDisable,
+			Advanced: true,
+		}, {
+			Name:     "multi_account_listing",
```

Comment:
These should be prefixed with `service_account` so they sort together and the user knows they are related

`service_account_multi_listing`

Reply:
> It is possible to rotate service account keys already over the API and using [rclone backend set](https://rclone.org/drive/#set) so is this code necessary?
When using a mount rotating the service account seems to have no effect when it comes to uploads. Might be outside the scope of this PR but would be nice if that worked.

Reply:
It works only if you use the rc command. The backend cli command doesn't work because it has no idea of which (if any or multiple) rclone to update the credentials on.

Reply:
> It works only if you use the rc command. The backend cli command doesn't work because it has no idea of which (if any or multiple) rclone to update the credentials on.

Did talk with darth at the time of this comment and couldn't find out why it did not work for me. After a chat today we managed to track down my particular issue (formatting of the fs parameter) and I therefore retract my previous statement about it not working (editing service account through remote control).